packages/d/dogtail/package.yml

```diff
@@ -13,9 +13,10 @@ builddeps  :
     - pkgconfig(python3)
 rundeps    :
     - pyatspi2
-    - python3-cairo
     - python-gobject
+    - python3-cairo
```

Comment:
`python3-cairo` can be removed from rundeps because it is dependency of `python-gobject`.